src/components/MessageView/MessageView.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState } from "react";
+import { ChatContext } from "modules";
+import styles from "./MessageView.module.css";
+import { BiSend } from "react-icons/bi";
+import { Message} from "components";
+
+export const MessageView = () => {
+  const [inputValue, setInputValue] = useState("");
+
+  const { container, header, wrapper, inputWrapper, input } = styles;
+  const { users, userMessagesID } = useContext(ChatContext);
+  const filterResult = users.filter(el => {
+    return el.id === userMessagesID
+  })
+  let mess;
+  let personName;
+  if (filterResult.length > 0) {
+    personName = filterResult[0].name
+    mess = filterResult[0].conversation.map(el => {
+      return <Message content={el.content} time={el.timestamp} status={el.status} key={Math.random()} />
```

Comment:
Never use JSX outside of scope of render() function or you need to do separated component for it
I suggest to just use it here in return (...) scope